dotcom-rendering/src/model/enhanceCommercialProperties.ts

```diff
@@ -15,8 +19,8 @@ const cappedDimensions = ({
 };
 
 const enhanceEditionCommercialProperties = (
-	properties: EditionCommercialProperties,
-): EditionCommercialProperties => {
+	properties: CommercialProperties[EditionId],
+): CommercialProperties[EditionId] => {
```

Comment:
Mapped types here to access `EditionCommercialProperties` without actually exporting it. This may seem like an indirection, and we could decide to export this type directly.

Reply:
My feeling is that exporting it directly would be 'simpler', given how we normally handle types in this codebase, and I think there's something to be said for simplicity here unless there are any real downsides. But I don't have a strong opinion on this.